src/components/views/settings/index.tsx

```diff
@@ -163,7 +178,21 @@ export const Settings: React.FC<SettingsProps> = ({navigation}) => {
                   accessibilityLabel={label}
                   accessibilityRole="button"
                   accessibilityHint={hint}
-                  onPress={() => navigation.navigate(screen)}>
+                  onPress={() =>
+                    screen === 'test'
+                      ? Linking.openURL(
+                          `mailto:support@example.com?subject=App Feedback for COVID Alert NY: Version ${version}&body=\n\n\n\n
+                          Device: ${getModel()}
+                          OS version: ${Platform.Version}
```

Comment:
The OS version should almost certainly include which OS we're talking about, like:

```
OS version: ${Platform.OS} ${Platform.Version}
```